pkgs/development/python-modules/pyro-ppl/default.nix

```diff
@@ -1,39 +0,0 @@
-{ buildPythonPackage, fetchPypi, lib, pytorch, contextlib2
-, graphviz, networkx, six, opt-einsum, tqdm }:
-buildPythonPackage rec {
-  version = "1.3.1";
-  pname = "pyro-ppl";
-
-  src = fetchPypi {
-    inherit version pname;
-    sha256 = "a034d9311d4715a2e8e127e0a4dd2996cbd34c4b85ac57b02b277c176b0a62ff";
-  };
-
-  propagatedBuildInputs = [
-    pytorch
-    contextlib2
-    # TODO(tom): graphviz pulls in a lot of dependencies - make
-    # optional when some time to figure out how.
-    graphviz
-    networkx
-    six
-    opt-einsum
-    tqdm
-  ];
-
-  # pyro not shipping tests do simple smoke test instead
-  checkPhase = ''
-    python -c "import pyro"
-    python -c "import pyro.distributions"
-    python -c "import pyro.infer"
-    python -c "import pyro.optim"
-  '';
-
-  meta = {
-    description = "A Python library for probabilistic modeling and inference";
-    homepage = "http://pyro.ai";
-    license = lib.licenses.mit;
-    maintainers = with lib.maintainers; [ teh ];
-    broken = true;
```

Comment:
This thing looks extremely actively maintained (last commit 1 hour ago).  @stites @drewrisinger either of you used it?  Maybe it was just broken due to an old pytorch, and now just needs an uprev.

Reply:
could be that it has a very large and fragile dependency graph, in which case, it would be hard to maintain in context on the nixpkgs python set

Reply:
I'm using actively but we're not on master nixpkgs. Might indeed be easier to maintain out of core nixpkgs.

Reply:
@FRidh might have an opinion because he updated to the recent version.

Reply:
looks like it was part of a large update, and pyro-ppl is "just a leaf" package, so it wasn't looked at for regressions

Reply:
@bhipple never used it.